__tests__/integrations/Dialog/CreateDialog.test.js

```diff
@@ -58,7 +58,7 @@ describe("Authentication POST /dialog", () => {
 });
 
 describe("Create Dialog", () => {
-  it("Check dialogue creation with all fields correctly filled.", async () => {
+  it("Check dialogue creation with all fields correctly fille d.", async () => {
```

Comment:
You don't need this change here, I believe it was a typo.

Use `Check dialogue creation with all fields correctly filled.`